frontend/src/scenes/events/eventsTableLogic.test.ts

```diff
@@ -40,16 +41,20 @@ const makePropertyFilter = (value: string = randomString()): PropertyFilter => (
 })
 
 const firstEvent = makeEvent('1', 'the first timestamp')
-const secondEvent = makeEvent('1', 'the second timestamp')
+const secondEvent = makeEvent('1', '2023-05-05T00:00:00.000Z')
 
-const baseEventsUrl = `api/projects/${MOCK_TEAM_ID}/events/`
-const beforeLastEventsTimestamp = '&before=the%20second%20timestamp'
-const afterTheFirstEvent = '&after=the%20first%20timestamp'
-const afterOneYearAgo = '&after=2020-05-05T00%3A00%3A00.000Z'
-const orderByTimestamp = '&orderBy=%5B%22-timestamp%22%5D'
-const propertiesWithFilterValue =
-    '?properties=%5B%7B%22key%22%3A%22fixed%20value%22%2C%22operator%22%3A%22exact%22%2C%22type%22%3A%22t%22%2C%22value%22%3A%22v%22%7D%5D'
-const emptyProperties = '?properties=%5B%5D'
+const beforeLastEventsTimestamp = '2023-05-05T00:00:00.000Z'
+const afterLastEventsTimestamp = '2023-04-30T00:00:00.000Z'
+const afterTheFirstEvent = 'the first timestamp'
+const afterOneYearAgo = '2020-05-05T00:00:00.000Z'
+const fiveDaysAgo = '2021-04-30T00:00:00.000Z'
+const orderByTimestamp = '["-timestamp"]'
+const propertiesWithFilterValue = '[{"key":"fixed value","operator":"exact","type":"t","value":"v"}]'
+const emptyProperties = '[]'
+
+const getUrlParameters = (url: string): Record<string, any> => {
+    return fromParamsGivenUrl('?' + (url.split('?').pop() || ''))
```

Comment:
this string manipulation feels like a signal that the `fromParamsGivenUrl` method needs a better API but no need to block this PR on that